BCStudentSoftwareDevTeam/celts
# Vs transcript interface #96

This PR fixes #96
- Create the front and back end for the service transcript for volunteers. 
- Create test for the transcript logic 

to test: 
- go to     /serviceTranscript
-  make sure it matches this page :  https://balsamiq-wireframes.appspot.com/?state={%22action%22:%22open%22,%22ids%22:[%221NWCXtb1Wvskk2VJNgxxk2lIl937hzC90%22],%22userId%22:%22113015963196064013745%22}
